models/MythoMax-L2-13B.json

```diff
@@ -0,0 +1,58 @@
+{
+    "_descriptorVersion": "0.0.1",
+    "datePublished": "2023-08-10T20:51",
+    "name": "MythoMax L2 13B",
+    "description": "An improved, potentially even perfected variant of MythoMix, MythoLogic-L2 and Huginn merge using a highly experimental tensor type merge technique. The main difference with MythoMix is that it was allowed more of Huginn to intermingle with the single tensors located at the front and end of a model, resulting in increased coherency across the entire model. This model is proficient at both roleplaying and storywriting due to its unique nature. Using MythoLogic-L2's robust understanding as its input and Huginn's extensive writing capability as its output seems to have resulted in a model that exceeds at both",
```

Comment:
Would be good to start off by introducing what this model "is good at" -- for folks who likely haven't heard of it before.  You can give the hugging face model page to GPT4 and ask for "a professional and technical summary, targeted at AI practitioners and users that keeps the important details" something along those lines.